templates/components/streamfield/blocks/stat_block.html

```diff
@@ -1,5 +1,9 @@
 {% verbatim %}
-{% include "components/streamfield/blocks/heading2_block.html" with large_spacing=True value=value.heading sr_only=value.sr_only_label %}
+{% if value.sr_only_label %}
+    <h2 class="sr-only">{{value.heading}}</h2>
+{% else %}
+    {% include "components/streamfield/blocks/heading2_block.html" with large_spacing=True value=value.heading %}
+{% endif %}
```

Comment:
Given that we need the same in `card_section_block.html` for #44, would it be simpler to update `heading2_block.html` to add something like `{% if sr_only %}sr-only{% endif %}` in the `<h2`>'s `class`?

It seems to be what the code originally intended, as this template already passed `sr_only=value.sr_only_label`.

Reply:
You're right. I updated the code to reflect that and I'll updated #44 as well to reflect this fix.